projects/angular-sdk/src/components/global.component.ts

```diff
@@ -499,9 +502,13 @@ export class GlobalComponent implements OnChanges {
             this.translateService.get('COMMON.ERROR_CODE.' + builtError.code)
             .pipe(take(1))
             .subscribe(errorDescription => {
+                errorMessage = errorDescription;
+
```

Comment:
il ne faudrait pas rajouter this.errorMessage = errorMessage; ici ? Etant donné qu'on est sur un callback ?